.github/test-infra/azure/aks/variables.tf

```diff
@@ -25,7 +25,7 @@ variable "dns_prefix" {
 
 variable "sku_tier" {
   description = "(Optional) The SKU Tier that should be used for this Kubernetes Cluster. Possible values are Free and Paid (which includes the Uptime SLA). Defaults to Free."
```

Comment:
Is this description wrong? `Possible values are Free and Paid` but we're setting `Standard` 👀

Reply:
Definitely incorrect, will research and provide appropriate values.